packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -98,36 +74,24 @@ if (!targetDir) {
 
 const newAppDir = path.resolve(process.cwd(), targetDir)
 const appDirExists = fs.existsSync(newAppDir)
+const templateDir = path.resolve(__dirname, '../template')
 
 if (appDirExists && fs.readdirSync(newAppDir).length > 0) {
   console.error(`'${newAppDir}' already exists and is not empty.`)
   process.exit(1)
 }
 
 const createProjectTasks = ({ newAppDir }) => {
-  const tmpDownloadPath = tmp.tmpNameSync({
-    prefix: 'redwood',
-    postfix: '.zip',
-  })
-
   return [
     {
       title: `${appDirExists ? 'Using' : 'Creating'} directory '${newAppDir}'`,
       task: () => {
-        fs.mkdirSync(newAppDir, { recursive: true })
-      },
-    },
-    {
-      title: 'Downloading latest release',
-      task: async () => {
-        const url = await latestReleaseZipFile()
-        return downloadFile(url, tmpDownloadPath)
+        fs.ensureDirSync(path.dirname(newAppDir))
+        if (!appDirExists) {
```

Comment:
We already check if the directory exists and is empty on line 79, but maybe the code will be more secure, and easier to understand if we move line 79-82 into this task instead?

Reply:
I understand why we did it previously because we didn't want to download and decompress the template if it was going to exit, but in this new structure that's not a concern.

Reply:
Got it. In that case, I'll remove this check and simply ensure that the parent directory exists before we do the copy.